source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -14,64 +14,121 @@ Overview
 --------
 
 In this guide, you can learn how to use the {+driver-short+} to
-perform **bulk operations**.
+perform **bulk operations**. Bulk operations help reduce the number
+of calls to the server. Instead of sending a request for each operation,
+you can perform multiple operations within one action.
 
-Bulk operations perform multiple write operations against one or more
-collections within a database. In MongoDB, you refer to a collection by combining
-the database name and the collection name in the format ``<database>.<collection>``. 
+.. tip::
 
-Bulk operations help reduce the number of calls to the server. Instead of sending
-a request for each operation, you can perform multiple operations within one action.
+   To learn more about bulk operations, see :manual:`Bulk Write Operations </core/bulk-write-operations/>`
+   in the {+mdb-server+} manual.
+
+You can use bulk operations to perform multiple write operations on
+a collection. You can also call bulk operations on your client,
```

Comment:
```suggestion
a collection. You can also execute bulk operations from the client,
```

Reply:
Fixed, but used "run" to [avoid "execute"](https://www.mongodb.com/docs/meta/style-guide/terminology/terms-for-global-audience/#use-jargon-with-care)